fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/BaseObject.java

```diff
@@ -255,7 +255,7 @@ public void applyTx(IDatabaseAdapter target, ITransactionProvider tp, IVersionHi
      */
     @Override
     public void applyVersion(IDatabaseAdapter target, IVersionHistoryService vhs) {
-        if (vhs.applies(getSchemaName(), getObjectType().name(), getObjectName(), version)) {
+        if (vhs.applies(getSchemaName(), getObjectType().name(), getObjectName(), version) || getObjectType().equals(DatabaseObjectType.PROCEDURE)) {
```

Comment:
wonder if we should add a comment to explaining the why.
basically we just want the procedures to get re-written every time (whether its a newer version or not)?

Reply:
Right.  I can add a comment